python/lsst/ip/isr/isrFunctions.py

```diff
@@ -134,6 +134,27 @@ def makeThresholdMask(maskedImage, threshold, growFootprints=1, maskName='SAT'):
     return measAlg.Defects.fromFootprintList(fpList)
 
 
+def growMasks(mask, radius=0, maskNameList=['BAD'], maskValue="BAD"):
+    """Grow a mask by an amount and add to the BAD plane.
+
+    Parameters
+    ----------
+    mask : `lsst.afw.image.Mask`
+        Mask image to process.
+    radius : scalar
+        Amount to grow the mask.
+    maskNameList : `list` [`str`]
```

Comment:
This isn't correct anymore since you no longer pass in a list here:
```
                    isrFunctions.growMasks(ccdExposure.getMask(),
                                           radius=self.config.brighterFatterMaskGrowSize,
                                           maskNameList=maskPlane,
                                           maskValue=maskPlane)
```
 You're passing in a single string.  The current interface isn't bad, but worth thinking about whether growing a maskNameList with a single maskValue is the most reusable.

Reply:
growMasks uses the mask.getPlaneBitMask() method, which accepts vectors of strings or single strings, and returns the appropriate value.  I've updated the docstring to note both `str` and `list` [`str`] are valid types.